website/docs/components/table/partials/code/component-api.md

```diff
@@ -138,6 +164,16 @@ If the `Th` component is passed as the first cell of a table body row, `scope="r
   </C.Property>
 </Doc::ComponentApi>
 
+### Table::ThSelectable
```

Comment:
@KristinLBradley I don't think we should expose this, it's an internal sub-component (similar to the `ThButtonSort` or `ThButtonTooltip`). Is it OK if I remove it?

Reply:
Yeah that's fine, thanks.